cmd/gaia/app/genesis.go

```diff
@@ -130,7 +117,7 @@ func GaiaAppGenTxNF(cdc *wire.Codec, pk crypto.PubKey, addr sdk.Address, name st
 
 	validator = tmtypes.GenesisValidator{
 		PubKey: pk,
-		Power:  freeFermionVal,
+		Power:  server.FreeFermionVal,
```

Comment:
as per earlier comment, should still be defined at the application level and passed into server

Reply:
as per earlier comment, it was reset to original form (not exported, local in genesis.go)